### ISSUE TITLE
web/images outputs appear to be entirely red images

### ISSUE THREAD

**Issue by pkmital** [NVIDIA/vid2vid#68]
Hi,

Thank you for a fascinating release.  I've tried a few epochs with my own dataset of images and finding the images produced during training have many fully red image outputs (example screenshot below), though these are not like the images in my dataset.  Is there something I could be doing wrong or is this expected behavior?  Please let me know if you need more information!

Current train command:

`python train.py --name cpvs --loadSize 512 --n_frames_total 30 --max_frames_per_gpu 2 --n_downsample_G 1 --num_D 1 --dataroot datasets/cpvs/ --n_gpus_gen -1 --print_freq 5 --niter 10000 --niter_decay 5000 --nThreads 4 --save_epoch_freq 1`

Example output:

<img width="1672" alt="screen shot 2018-11-01 at 12 54 40 am" src="https://user-images.githubusercontent.com/429252/47839668-bfdff000-dd70-11e8-88a5-657b2f93a6d3.png">

**Comment by tcwang0509** [comment]
It's definitely not expected. Is your real image 3-channel?

**Comment by pkmital** [comment]
Yes both A & B are 3 channel images.  Attaching a pair for reference (note: they are different sizes, is this an issue?)

one image of a sequence in A:

![00000090](https://user-images.githubusercontent.com/429252/47943233-48b27500-deb2-11e8-829f-af36954fe3ce.png)

one image of a sequence in B:

![00000090](https://user-images.githubusercontent.com/429252/47943246-58ca5480-deb2-11e8-8779-4e9bab2e29f9.png)
